### PR TITLE
[FIX] tools: Return input_str when removing accents if empty

### PR DESCRIPTION
**Pull request by nboulif**
Issue

	remove_accents(input_str) return a string 'False' if the param
	is a boolean.

Solution

	Return input_str if equal '' or False.

opw-2278959

### CODE REVIEW

**Comment by nboulif** [comment]
@nim-odoo I did some test an put the results in comments. Did not find a specific file where to do this kind of unitest, any advice?

**Comment by nim-odoo** [comment]
https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/tests/test_misc.py should be fine.

**Comment by nboulif** [comment]
Thanks, I missed it. Done :+1:

File: odoo/tools/misc.py [hunk 1]
```diff
@@ -741,7 +741,22 @@ def g(obj):
 def remove_accents(input_str):
     """Suboptimal-but-better-than-nothing way to replace accented
     latin letters by an ASCII equivalent. Will obviously change the
-    meaning of input_str and work only for some cases"""
+    meaning of input_str and work only for some cases
+
+    Examples::
```
Comment (nim-odoo) [inline odoo/tools/misc.py:746]: We can remove the examples now :)

Comment (nboulif) [reply odoo/tools/misc.py]: Done :+1:

**Pull request closed without merging**